R/rectangle.R

```diff
@@ -722,8 +732,25 @@ elt_to_wide <- function(x, name, strict, names_sep) {
   x
 }
 
+#' Automatically call `unnest_wider()` or `unnest_longer()`
+#'
+#' @description
+#' `unnest_auto()` picks between `unnest_wider()` or `unnest_longer()`
+#' by inspecting the inner names of the list-col:
+#'
+#' * If all elements are unnamed, it uses
+#'   `unnest_longer(indices_include = FALSE)`.
+#' * If all elements are named, and there's at least one name in
+#'   common across all components, it uses `unnest_wider()`.
+#' * Otherwise, it falls back to `unnest_longer(indices_include = TRUE)`.
+#'
+#' It's handy for very rapid interactive exploration but I don't recommend
+#' using it in scripts, because it will succeed even if the underlying data
+#' radically changes.
```

Comment:
I think you need to redocument again because this section didn't show up for me in the doc page